src/lib.rs

```diff
@@ -331,6 +346,15 @@ where
         .author(clap::crate_authors!())
         .about(clap::crate_description!())
         .setting(AppSettings::SubcommandRequiredElseHelp)
+        .subcommand(
+            SubCommand::with_name("debug")
+                .about("Print the tmux commands that would be used to start and configure a tmux session using a path to a project config file")
+                .arg(
+                    Arg::with_name("PROJECT_CONFIG_FILE")
+                        .help("The path to the project config file")
+                        .required(true),
```

Comment:
Oooo, since this `Arg` is the same one that's used in `start` we can actually use one of the strong points of `clap` where you can define the common `Arg` above the `App` and then give both subcommands a reference to the common arg. This helps keep the command line arg parsing DRY.

Reply:
Neat. I'll investigate this.